docker/env.sh

```diff
@@ -41,6 +51,14 @@ REPOSITORY_NAME="${REPOSITORY_NAME-$(basename "$(git rev-parse --show-toplevel)"
 REPOSITORY_NAME="${REPOSITORY_NAME,,}"
 VOLUMENAME="${VOLUMENAME-"${REPOSITORY_NAME}_data"}"
 ARCH="${ARCH-$(uname -m)}"
+if [ $ARCH == "aarch64" ]
+then
+   ARCH=arm64
+fi
+if [ $ARCH == "x86_64" ]
```

Comment:
I had no problems with neither aarch64 nor x86_64 (tested on a M1 and a I7)

Reply:
Not sure why but both could not find the repository when I tried those arch names until I changed it to the ones displayed on the docker hub site.  Also, on podman you have to specify docker.io fwiw

Reply:
the missing docker.io in the base image tag is totally my fault and the dockerfile is much "cleaner" with the registry preponed to the base-image tag 😅

Reply:
heh sounds good